cvat/schema.yml

```diff
@@ -6152,6 +6771,53 @@ components:
         * `annotation` - ANNOTATION
         * `validation` - VALIDATION
         * `completed` - COMPLETED
+    JobType:
+      enum:
+      - annotation
+      - ground_truth
+      type: string
+      description: |-
+        * `annotation` - ANNOTATION
+        * `ground_truth` - GROUND_TRUTH
+    JobWriteRequest:
+      type: object
+      properties:
+        assignee:
+          type: integer
+          nullable: true
+        stage:
+          $ref: '#/components/schemas/JobStage'
+        state:
+          $ref: '#/components/schemas/OperationStatus'
+        type:
+          $ref: '#/components/schemas/JobType'
+        task_id:
+          type: integer
+        frame_selection_method:
+          $ref: '#/components/schemas/FrameSelectionMethodEnum'
+        count:
+          type: integer
+          minimum: 0
+          description: |
+            The number of frames included in the job.
+            Applicable only to the random frame selection
+        seed:
+          type: integer
+          minimum: 0
+          description: |
+            The seed value for the random number generator.
+            The same value will produce the same frame sets.
+            Applicable only to the random frame selection
```

Comment:
This should document what happens when the seed is unspecified.

Reply:
Added